src/main/java/core/basesyntax/impl/StorageImpl.java

```diff
@@ -3,17 +3,94 @@
 import core.basesyntax.Storage;
 
 public class StorageImpl<K, V> implements Storage<K, V> {
+    private static final int DEFOULT_SIZE = 10;
+    private Pairs[] pairs;
+    private Pairs[] massiv;
+    private int index = 0;
```

Comment:
```suggestion
    private Pairs<K, V>[] pairs;
    private int size;
```
you don't need 2 arrays, 
lets store the current size, this variable also will be the next empty index